apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/TransactionImpl.java

```diff
@@ -346,6 +375,12 @@ private TransactionImpl(BuilderImpl builder, byte[] keySeed) throws AplException
 
     }
 
+    private void lookupAndInjectBlockchain() {
```

Comment:
Maybe better here also return injected value to make it possible to use blockchain in one line:  `lookupAndInjectBlockchain().getHeight()`

Reply:
OK, I'll do that.